src/date.ml

```diff
@@ -476,7 +476,7 @@ value string_of_ondate conf d =
 value string_of_date conf =
   fun
   [ Dgreg d _ -> string_of_dmy conf d
-  | Dtext t -> "(" ^ t ^ ")" ]
+  | Dtext t -> t ]
```

Comment:
Yes, there are two other places where parenthesis are added to textual dates. Ideally, parenthesis should be added in the template, or in the calling code if not produced by a template. To facilitate the test for textual date presence, I suggested adding a %date.text; function (similar to %date.day;) which returns the textual date or "" .